### ISSUE TITLE
Security Policy Module?

### ISSUE THREAD

**Issue by jake2184** [fortinet-ansible-dev/ansible-galaxy-fortimanager-collection#16]
Hi,

I'm looking to use Ansible to drive creation of Security Policies. However, I cannot find a module in this collection which does that. 

fmgr_pkg_firewall_policy is very close, and has the arguments I would expect, but pushes to "firewall policy" rather than "firewall security-policy".

Am I missing something? If the module doesn't exist, why not?

Thanks,

Jake

**Comment by chillancezen** [comment]
hi Jake, 
is this one what you are looking for https://ansible-galaxy-fortimanager-docs.readthedocs.io/en/galaxy-2.0.1/daemon_docgen/fmgr_securityconsole_install_package.html?

it pushes policy packages to a device.

**Comment by jake2184** [comment]
No, I was looking for a module which creates security policies in the Manager.
